extensions/adobe/experience/decisioning/placement-details.schema.json

```diff
@@ -0,0 +1,76 @@
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/placement-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Decision Placement Detail",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Detail about the entity defining and controlling the decisioning process. Decisions are made in the context of a placement. At any given time, the best option is re-evaluated, or ranked, based on the most current set of context variables, rules and constraints applicable for this placement.",
+  "definitions": {
+    "placement-details": {
+      "properties": {
+        "https://ns.adobe.com/experience/decisioning/name": {
+          "title": "Placement Name",
+          "type": "string",
+          "description": "An assigned name for the placement to refer to it in human interactions"
+        },
+        "https://ns.adobe.com/experience/decisioning/description": {
+          "type": "string",
+          "title": "Placement Description",
+          "description": "Placement description. It is used to convey human readable intentions on how dynamic content is used in the overall message delivery. That a certain space is a \"Banner\" in a web page is often conveyed via the description and not by a formal method."
+        },
+        "https://ns.adobe.com/experience/decisioning/channelID": {
+          "title": "Placement's Channel Identifier",
+          "type": "string",
+          "format": "uri-reference",
+          "description": "The channel in which proposition was made. The value is a valid Channel URI. See 'https://ns.adobe.com/xdm/channels/channel'"
+        },
+        "https://ns.adobe.com/experience/decisioning/componentType": {
+          "$ref": "https://ns.adobe.com/experience/decisioning/content-component-details#/definitions/content-component-type",
+          "title": "Content Component Type",
+          "description": "A constraint for the content that can be shown in the place described by this placement. Component Types for example are: image link, html block or plain text."
+        },
+        "https://ns.adobe.com/experience/decisioning/contentTypes": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "title": "MIME Media Type",
+            "description": "[Multipurpose Internet Mail Extensions (MIME)](https://tools.ietf.org/html/rfc2046)",
+            "pattern": "\\w+\\/[-.\\w]+(?:\\+[-.\\w]+)?",
+            "description": "A constraint for the media type of the components that is expected in that placement. There could be more than one media type possible for one component such as different image format."
+          }
+        }
+      }
+    },
+    "placement-snapshot-identifier": {
+      "properties": {
+        "https://ns.adobe.com/experience/decisioning/id": {
```

Comment:
If this kind of id is defined in multiple schemas and within different definitions, can we give them a sematic name like placemantID ? Also, "id" should be upper case "ID". This comment applied to all the IDs defined in this PR. Could you please check ?